src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/TypeAccessibilityCodeActionProvider.cs

```diff
@@ -121,10 +140,10 @@ internal class TypeAccessibilityCodeActionProvider : CSharpCodeActionProvider
             {
                 fqn = codeAction.Title;
             }
+            // When there are multiple FQN suggestions, the code action title is of the form:
+            // `Fully qualify 'Dns' -> System.Net.Dns`
             else
             {
-                // When there are multiple FQN suggestions, the code action title is of the form:
-                // `Fully qualify 'Dns' -> System.Net.Dns`
                 var expectedCodeActionPrefix = $"Fully qualify '{associatedValue}' -> ";
```

Comment:
how does this work for non-english?

Reply:
So https://github.com/dotnet/aspnetcore/issues/25939 will get the identifiers from Roslyn/VS, however this specific code block is VSCode specific. VSCode is breaking down multiple CodeAction.Children into the `Fully qualify '<TYPE>' -> '<FQN_TYPE>'` code actions. I'm not entirely sure how O#/VSCode handles localization to that respect. This issue will be resolved when VSCode/O# code action handling supports the `textDocument/codeAction` & `textDocument/codeActionResolve` endpoints.